infra/terraform/modules/data_access/gh_webhook_handler/index.py

```diff
@@ -0,0 +1,91 @@
+import hmac
+from hashlib import sha1
+import json
+import os
+
+import boto3
+from botocore.exceptions import ClientError
+
+
+SNS_ARN = os.environ.get("SNS_ARN")
+STAGE = os.environ.get("STAGE")
+GH_HOOK_SECRET = os.environ.get("GH_HOOK_SECRET")
+
+sns = boto3.client("sns")
+
+
+class SNSTopicNotFoundError(Exception):
```

Comment:
The Python documentation is wrong in this case, I think. `NotFound` already conveys the fact that it is an error. They are referring to something like `SNSTopicError`, for example.